sda_download/sda_download.go

```diff
@@ -154,6 +178,58 @@ func datasetCase(token string) error {
 	return nil
 }
 
+func recursiveCase(token string) error {
+	fmt.Println("Downloading content of the path(s)")
+	// get all the files of the dataset
+	files, err := getFilesInfo(*URL, *datasetID, "", token)
+	if err != nil {
+		return err
+	}
+	// check all the provided paths and add a slash
+	// to each one of them if does not exist and
+	// append them in a slice
+	var dirPaths []string
+	for _, path := range Args.Args() {
+		if !strings.HasSuffix(path, "/") {
```

Comment:
The user is not forced to add the "/" for the recursive case. The code is just checking if it is there and if it is not then it is added because if it is not then later when the loop takes place there is a strings.Contains which will download maybe something that it should not, eg: if the folder path provided is `main/folder` and then there is another one named `main/folder-new.` In that case it will download the content of both folders